pkg/config/loader.go

```diff
@@ -79,6 +80,12 @@ func LoadConfig() (*Config, error) {
 		cfg.ResultsDir = resultsDir
 	}
 
+	// If the loaded config doesn't have its own UUID, create one
```

Comment:
in the old version of sonobuoy, the UUID is generated by `cfg := New()`; and in the current version the `cfg := &Config{}`; is it better to just use `cfg := New()`, so that all the default values, not only the UUID will be generated? Or it can break some other logic?

Reply:
In the [PR which introduced this change](https://github.com/heptio/sonobuoy/pull/725), it seems that the intent was to prevent other default values being introduced, particularly `Resources`. It looked to me like the setting of `UUID` was something that was missed in that change. Hopefully, @johnSchnake can give us a bit more insight into the intended behaviour here when he's back from vacation.